docs/modules/ROOT/pages/plugins.adoc

```diff
@@ -0,0 +1,25 @@
+= Relevant Plugins
+
+This is a list of relevant plugins providing extra functionality to Nile.
+
+TIP: If you are a plugin developer and your project is not listed, reach out to us!
```

Comment:
how? this call to action is not properly set. should we add a link to opening a PR? should we have a PR template for docs?